rules/jsonschema-2019-09-to-2020-12.json

```diff
@@ -10,7 +10,8 @@
         "required": [ "$schema" ],
         "properties": {
           "$schema": {
-            "const": "https://json-schema.org/draft/2019-09/schema"
+            "type":"string",
+            "pattern": "http[s]?://json-schema.org/draft/2019-09/schema[#]?"
```

Comment:
This pattern matches the given URL in a wider string. You should limit the regex with `^` and `$`